exec-agent/src/term/server.go

```diff
@@ -38,13 +38,29 @@ import (
 	"os/exec"
 	"sync"
 	"unicode/utf8"
+	"syscall"
 )
 
 type wsPty struct {
 	Cmd     *exec.Cmd // pty builds on os.exec
 	PtyFile *os.File  // a pty is simply an os.File
 }
 
+func (wp *wsPty) Stop(finalizer *ReadWriteRoutingFinalizer) {
+	closeFile(wp.PtyFile, finalizer)
```

Comment:
Please comment this function well as it is not obvious and we will forget why it is implemented in such a way